tests/api/cluster/test_group.py

```diff
@@ -0,0 +1,29 @@
+from model_mommy import mommy
+import pytest
+
+from controlpanel.api import cluster
+
+
+@pytest.fixture(autouse=True)
+def enable_db_for_all_tests(db):
+    pass
```

Comment:
I've seen this in several tests files, is this a special method that once defined tells pytest to not write to the DB or something?

Reply:
It tells `pytest-django` that the test method is allowed to access the database, which is disabled by default.